groups/frontend/data.tf

```diff
@@ -99,6 +99,10 @@ data "aws_network_interface" "nlb" {
   }
 }
 
+data "vault_generic_secret" "build_subnet_cidrs" {
+  path = "aws-accounts/shared-services/build_subnet_cidrs"
+}
+
```

Comment:
It may be better to utilise the `shared-services-management-cidrs` prefix list - this should be available for use in all main accounts.

Reply:
Good shout. 👍🏻